### ISSUE TITLE
Submissions that are not complete should not be displayed to RPE judges

### ISSUE THREAD

**Issue by dboyer** [Iridescent-CM/technovation-app#3847]
Currently if a submission is assigned to an RPE, it appears in the list of projects for a judge to review even if the submission is not complete. We should not allow submissions in progress to appear on the judge pages.

For example, this is the team list for the event Madrid Senior. 
<img width="1061" alt="Screen Shot 2023-02-15 at 12 16 47 PM" src="https://user-images.githubusercontent.com/516764/219103927-9cdac4a1-4e82-47c9-b8be-bbc900c5b396.png">

The submission is available for this judge to review even though it is still in progress. Sections of the submission will not load such as the technical questions since the participant has not selected the type of project (app or AI) that they built. 
<img width="811" alt="Screen Shot 2023-02-15 at 12 17 56 PM" src="https://user-images.githubusercontent.com/516764/219104330-a1987492-f8af-41e9-8a8e-275b0ce7909c.png">

<img width="1213" alt="Screen Shot 2023-02-13 at 11 57 02 AM" src="https://user-images.githubusercontent.com/516764/219105242-cebcff7e-fdec-4e87-93a3-2b39b5b8c9e4.png">

**Comment by dboyer** [comment]
Program team confirmed that incomplete submissions should not be visible on the platform to RPE judges.

**Comment by dboyer** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Technovation-605271851956b1000ec2f24d/issues/Iridescent-CM/technovation-app/3847?planning-poker) with Zenhub @shaunxp20 @viviancan

**Comment by dboyer** [comment]
First step is to limit submissions that appear on RPE judge interface based on "is eligible." There may be additional filters/limits put in that would not allow ChAs to assign an incomplete submission to judges.

**Comment by dboyer** [comment]
One note on this. I'm pretty sure that even if a submission is assigned to a particular judge as part of an RPE, it is still visible to other judges. 

On QA, the event Madrid Senior includes a submission for Barcelona Bulldogs that is assigned to a specific judge called "Deb Spanish Judge." I then logged in as Madrid Judge 2 another judge attending that event who did not have specific submissions assigned. Madrid Judge 2 also had Barcelona Bulldogs in the list of submissions to judge:

![Screen Shot 2023-03-01 at 10.49.16 PM.png](https://images.zenhubusercontent.com/5ca619b412380971eb9c9477/520192e6-6b87-439d-9dd2-b8bc83e3fb4e)

**Comment by dboyer** [comment]
Confirming that on QA I only see the submissions for an RPE that are fully complete. I logged in as judge for the Paris Power Pitching Event, which has 3 complete submissions and 2 incomplete. I saw only the 3 complete submissions in my list of things to judge.

One note: if a judge was in the process of judging an incomplete submission (like the Deb Spanish Judge example in this issue), the incomplete submission still remains in their "scores in progress" section. I think that's okay though as we'll have this change up before judging starts so no one should be able to get in a state where they have half-judged incomplete submissions.
